pkg/network/setup/network.go

```diff
@@ -84,8 +84,8 @@ func (v VMNetworkConfigurator) getPhase1NICs(launcherPID *int, networks []v1.Net
 			return nil, fmt.Errorf("no iface matching with network %s", networks[i].Name)
 		}
 
-		// SR-IOV devices are not part of the phases
-		if iface.SRIOV != nil {
+		// Binding plugin and SR-IOV devices are not part of the phases
+		if iface.Binding != nil || iface.SRIOV != nil {
```

Comment:
This may not work well when we'll need to work with tap devices (or other future exceptions).
But these thoughts are futuristic, for now this seem fine.